content/en/docs/spinnaker/terraform-enable-integration.md

```diff
@@ -256,14 +256,59 @@ spec:
       armory:
         terraform:
           enabled: true
+          # Remote backends is an Early Access feature. Do not use it in production. Omit this field if you do not want to use remote backends.
+          remoteBackendSupport: true
+    profiles:
+      deck:
+        # Enables the UI for the Terraform Integration stage
+        settings-local.js: |
+          window.spinnakerSettings.feature.terraform = true;
 ```
 
+This example manifest also enables the Terraform Integration UI and support for remote backends.
+
 **Halyard**
 
 ```
 hal armory terraform enable
 ```
 
+### Remote backends
+
+{{< include "experimental-feature.html" >}}
+
+This Early Access feature is available in Armory 2.AB.XY and later.
+
+The Terraform Integration supports using remote backends provided by Terraform Cloud and Terraform Enterprise.
+
+When using remote backends, keep the following in mind:
+
+* The Terraform stage must use the same Terraform version that your Terraform Cloud/Enterprise workspace runs.
+* The minimum supported Terraform version is 0.12.0.
+* If you view a `plan` action in the Terraform Cloud/Enterprise UI, the type of `plan` action that the Terraform Integration performs is a "speculative plan." For more information, see [Speculative Plans](https://www.terraform.io/docs/cloud/run/index.html#speculative-plans).
+* You cannot save and apply a plan file.
+* You cannot use Named Profiles for authentication for remote backends.
```

Comment:
Small caveat here - we've added a `tfc` Named Profile kind that provides authentication support to the `remote` backend.

Reply:
Need to add this. Will work on it this week.